<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>708</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MSMonitor v0.8.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="minimumSize">
       <size>
        <width>1050</width>
        <height>600</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
    <addaction name="separator"/>
    <addaction name="action_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menu_Curves">
     <property name="title">
      <string>&amp;Curves</string>
     </property>
     <addaction name="action_Power"/>
     <addaction name="action_Temperature"/>
     <addaction name="action_Clock"/>
     <addaction name="action_Utilization"/>
     <addaction name="action_Memory"/>
    </widget>
    <widget class="QMenu" name="menu_Heatmaps">
     <property name="title">
      <string>&amp;Heatmaps</string>
     </property>
     <addaction name="action_Utilization_2"/>
     <addaction name="action_Temperature_2"/>
    </widget>
    <addaction name="menu_Curves"/>
    <addaction name="menu_Heatmaps"/>
    <addaction name="action_Systemoverview"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Systemoverview">
   <property name="text">
    <string>&amp;System Overview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Power">
   <property name="text">
    <string>&amp;Power</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Temperature">
   <property name="text">
    <string>&amp;Temperature</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_Clock">
   <property name="text">
    <string>&amp;Clock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Utilization">
   <property name="text">
    <string>&amp;Utilization</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_Memory">
   <property name="text">
    <string>&amp;Memory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_Utilization_2">
   <property name="text">
    <string>&amp;Utilization</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
  </action>
  <action name="action_Temperature_2">
   <property name="text">
    <string>&amp;Temperature</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
